### DISCUSSION TITLE
Config option

### DISCUSSION THREAD

**Discussion by jorgemmsilva** [renovatebot/renovate#21605] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

goGetDir option gets picked up and used

### What you tried so far.

`"goGetDirs"` option was added recently in this PR: https://github.com/renovatebot/renovate/pull/20156

However, after adding the `"goGetDirs"` options to `renovate.json`:
https://github.com/iotaledger/wasp/blob/a4cb5cb5b97ae949d9961e85019bec22abe7dc2b/renovate.json#L15-L18

I got this issue:
https://github.com/iotaledger/wasp/issues/2359

```
Error type: The renovate configuration file contains some invalid settings
Message: Invalid configuration option: goGetDirs
```

Am I missing something? Maybe the GH app is not up to date?

**Comment by rarkins** [comment, marked as answer]
The Renovate App does not deploy new versions immediately

**Reply by jorgemmsilva** [reply]
thanks, that makes sense :+1: 

when the GH app is updated, will the issue resolve itself, or does the config need to be changed?

**Reply by rarkins** [reply]
Yes, it will automatically resolve
